lib/rules/quotes.js

```diff
@@ -28,6 +40,7 @@ export default class Quotes extends Rule {
       [
         '  * "double" - requires the use of double quotes wherever possible',
         '  * "single" - requires the use of single quotes wherever possible',
+        '  * { hbs: `single`|`double`, html: `single`|`double` } - requires different quotes for Handlebars and HTML syntax',
```

Comment:
For the record, what do you think the ideal default should be for each type? So we can consider it in the nexst major version.

Reply:
I think it should be double quotes HTML attributes and single everything else. That's how most code I see is written. The only reason to go with double always is because it is the default prettier does, however most people seem to want it changed to single with this issue being very unclear on weather it will change or not. https://github.com/prettier/prettier/issues/4102